sdk/playwrighttesting/microsoft-playwright-testing/src/common/entraIdAccessToken.ts

```diff
@@ -37,6 +41,7 @@ class EntraIdAccessToken {
       );
       return true;
     } catch (err) {
+      process.stdout.write("\nFailed to fetch entraID access token");
```

Comment:
Similarly here, let's omit all error messages which are not mentioned in the doc.